pelican/readers.py

```diff
@@ -268,7 +268,11 @@ def _parse_metadata(self, meta):
                 formatted_values = "\n".join(value)
                 # reset the markdown instance to clear any state
                 self._md.reset()
+                # process field as markdown, but *don't* extract md metadata
+                preprocessors = self._md.preprocessors
+                self._md.preprocessors.pop('meta', None)
                 formatted = self._md.convert(formatted_values)
+                self._md.preprocessors = preprocessors
```

Comment:
Thanks for the feedback, @avaris. I could change this up a couple different ways:

1. Remove the meta preprocessor from `self._md` after the initial `self._md.convert` call
2. Create an alternate Markdown parser without the meta extension and use that for processing fields (using `copy.deepcopy` on `self.settings['MARKDOWN']`)

@justinmayer, do you have a preference?